.github/workflows/generate.yaml

```diff
@@ -30,9 +30,12 @@ jobs:
           repository: googleapis/discovery-artifact-manager
           fetch-depth: 1
           path: discovery-artifact-manager
-      - uses: actions/setup-python@v4
-        with:
-          python-version: 2.7.18
+      - run: |
+          sudo apt update
+          sudo apt install python2
+          python2 --version
+          curl https://bootstrap.pypa.io/pip/2.7/get-pip.py -o get-pip.py

```

Comment:
I tried `apt-get install python-pip` but it was not available (may need a source to be manually added).
`ensure-pip` was "disabled in this version of Ubuntu"